app/ide-desktop/lib/client/src/index.ts

```diff
@@ -114,14 +116,11 @@ class App {
         }
     }
 
-    /** Set Chrome options based on the app configuration. For comprehensive list of available
+    /** Sets Chrome options based on the app configuration. For comprehensive list of available
```

Comment:
In all of our Rust codebase we have the opposite rule - instead of "Sets" we use "Set". I think we should be consistent here. Using docs in the imperative form rather than 3rd person form makes them shorter and unified with Rust codebase (which is unified with official Rust style guide).

Reply:
ah - i think the convention for TS in the other files was to use the opposite, but should be easy enough to change back